### PR TITLE
Sign up logic.

### PR DESCRIPTION
**Pull request by jiv**
- Added SelectInput component.
- Updated SignUp Component with the required fields.
- Renamed some files for better clarity.

### CODE REVIEW

File: src/components/Signup/SignupForm.js [hunk 1]
```diff
@@ -40,6 +41,8 @@ const validate = values => {
   return (errors ? errors : {});
 };
 
+const allGenders = [{key:"female", value:"Female"}, {key:"male", value:"Male"}];
```
Comment (MaicolBen) [inline src/components/Signup/SignupForm.js]: Use single quotes

Comment (bernabe9) [inline src/components/Signup/SignupForm.js]: This should be in a js file with all the constants

File: src/components/Signup/SignupForm.js [hunk 2]
```diff
@@ -40,6 +41,8 @@ const validate = values => {
   return (errors ? errors : {});
 };
 
+const allGenders = [{key:'female', value:'Female'}, {key:'male', value:'Male'}];
```
Comment (bernabe9) [inline src/components/Signup/SignupForm.js]: This should be in a js file with all the constants.

Also, leave a blank space after start and end the brackets `{` `}`

File: src/components/common/SelectInput.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React, {PropTypes} from 'react';
```
Comment (bernabe9) [inline src/components/common/SelectInput.js]: space into the brackets

File: src/components/common/SelectInput.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+const SelectInput = ({input, label, defaultOption, error, options}) => {
```
Comment (bernabe9) [inline src/components/common/SelectInput.js]: same here

**Pull request merged**